frontend/src/createStream/SelectDeviceDialog/__tests__/SelectDeviceTable.test.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { afterEach, describe } from "@jest/globals";
+
+import SelectDeviceTable from "../SelectDeviceTable";
+
+import SelectTable from "../SelectTable";
+import DeviceInfo from "../../../model/DeviceInfo";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<SelectDeviceTable/> functional component", () => {
+  let wrapper;
+
+  describe("returns a component that", () => {
+    const dummyDevice1 = new DeviceInfo(
```

Comment:
could write this the same way your wrote `dummyDevice2` since you only need the name field